### ISSUE TITLE
Support for multi-source MySQL replication

### ISSUE THREAD

**Issue by ByEmke** [percona/percona-monitoring-plugins#25]
As of 5.7 MySQL started supporting multi-source replication. 

E.g commands:

```
show slave status for channel 'mymaster1'\G
show slave status for channel 'mymaster2'\G
```

Is it something you are considering to add?

**Comment by roman-vynar** [comment]
There is an option to work with MariaDB multi-source replication
```
    --master-conn NAME  Master connection name for MariaDB multi-source replication.
```
It runs `SHOW SLAVE <NAME> STATUS`. However, it looks like to be different than 5.7.

**Comment by ByEmke** [comment]
Yep, the syntax is different and you can't run `SHOW SLAVE <NAME> STATUS` on MySQL :).

Just wanted a heads up whether you are aware of it and are planning to update

**Comment by roman-vynar** [comment]
👍

**Comment by grypyrg** [comment]
implemented in #54
